src/Discord.Net.Core/Entities/Messages/AllowedMentionTypes.cs

```diff
@@ -8,17 +8,24 @@ namespace Discord
     [Flags]
     public enum AllowedMentionTypes
     {
+        /// <summary>
+        ///     No flag was set.
+        /// </summary>
+        /// <remarks>
+        ///     It is not used to control mentions and does not mean no mentions will be allowed.
+        /// </remarks>
+        None        = 0,
```

Comment:
I believe `Default` would be a better name in this case. And the summary could be something along the lines of `The default set of Allowed Mentions` (which just happens to be all, iirc).

Reply:
The default behavior (`new AllowedMentions()` or `new AllowedMentions(AllowedMentionTypes.None)`) is none (because an empty `parse` array means no mentions allowed), so, while not intentionally trying to, `AllowedMentionTypes.None` would be none if there's no OR.

Reply:
Oh yeah, my b. In that case `None` is fine.